env/.env.development

```diff
@@ -0,0 +1,2 @@
+# REACT_APP_FHIR_BASE_URL=https://localhost:9443/fhir-server/api/v4
```

Comment:
I would move these files to the top level which is the conventional place to put them.

Reply:
Done